accessibility/index.html

```diff
@@ -0,0 +1,550 @@
+                        <p>Once VoiceOver is Activated</p>
+						<p class="fragment">Activate an Item on the Screen: Double Tap</p>
+						<p class="fragment">Next Item: Flick Right</p>
+						<p class="fragment">Previous Item: Flick Left</p>
+						<p class="fragment">Scroll Down/Up: Three Finger Up/Down</p>
+						<p class="fragment">Activate Rotor: Turn Clock-wise</p>
+						<p class="fragment">Next/Previous Rotor Item: Flick Down/Flick Up</p>
+					</section>
+                    <aside class="notes">
+				    Instructions for using VoiceOver on an OSX device once it has been enabled. - Demo and Group Activity
+                    </aside>
+				</section>
+				
+				<!-- HOW TO USE VOICEOVER FOR IOS END -->
+                        
+                <!-- HOW TO ACTIVATE VOICEOVER FOR OSX START -->
+                        
+                <section>
+						<h2>Activate <span class="gdi">VoiceOver</span><br/>Screen Reader - OSX</h2>
+                    <small><span class="gdi">Instructions</span> for activating VoiceOver for OSX.</small>
+                    <p><strong><span class="gdi">Activate</span> VoiceOver</strong> from <span class="gdi">System Preferences</span> > <span class="gdi">Accessibility</span> > <span class="gdi">Enable VoiceOver</span> checkbox</p>
+				</section>
+                <aside class="notes">
+				    Instructions for using VoiceOver on an OSX device once it has been enabled. - Demo and Group Activity
+                </aside>
+                        
+                <!-- HOW TO ACTIVATE VOICEOVER FOR OSX END -->
+                        
+				<!-- IMPORTANCE OF SEMANTIC HTML START -->
+				
+				<section>
+				    <h2>Importance of</br><span class="gdi">Semantic HTML Elements</span></h2>
+                    <small>The accessibility API is part of a device's operating system and represents objects in a user interface, exposing information about each object within the application or Webpage</small>
+
+				    <p>Semantic HTML elements are immediately identified by the accessibility API and interpreted by screen readers. Non-semantic elements convey no information to the accessibility API.</p>
+                    <small>OSX Accessibility API Demo</small>
+                
+				</section>
+				
+				<!-- IMPORTANCE OF SEMANTIC HTML END -->
+                        
+                <!-- EXAMPLES OF SEMATIC VS NON-SEMANTIC HTML START -->  
+                    
+				<section>
+                    <h2><span class="gdi">Semantic</span> Elements</h2>
+                    <pre><code data-trim contenteditable> 
+                    
+<!-- Link Example -->
+ <a href="http://www.someurl.com">Go to some Web page</a>
+
+ <!-- Button Example -->
+ <button>Click me</button>
+ 
+                    </code></pre>
+                    <br/>
+                    <h2><span class="gdi">Non-Semantic</span> Elements</h2>
+                    <small>Divs styled with CSS to look semantic</small>
```

Comment:
- Added slide that explains how exercises illustrate how semantic HTML
elements and non-semantic elements can look the same but are interpreted
differently by screen readers

- Made JSbin public.

Thanks!

On Sun, Oct 4, 2015 at 12:03 PM, Brenda Jin <notifications@github.com>
wrote:

> In accessibility/index.html
> <https://github.com/pamelafox/teaching-materials/pull/169#discussion_r41101043>
> :
>
> > +                <!-- EXAMPLES OF SEMATIC VS NON-SEMANTIC HTML START -->
> > +
> > +				<section>
> > +                    <h2><span class="gdi">Semantic</span> Elements</h2>
> > +                    <pre><code data-trim contenteditable>
> > +
> > +<!-- Link Example -->
> > + <a href="http://www.someurl.com">Go to some Web page</a>
> > +
> > + <!-- Button Example -->
> > + <button>Click me</button>
> > +
> > +                    </code></pre>
> > +                    <br/>
> > +                    <h2><span class="gdi">Non-Semantic</span> Elements</h2>
> > +                    <small>Divs styled with CSS to look semantic</small>
>
> Can you provide a visual example of how the same button might look?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pamelafox/teaching-materials/pull/169/files#r41101043>
> .
>